changelog.yaml

```diff
@@ -37,6 +37,16 @@
     deprecations: 
     pullRequestNumber: 162
     jiraIssueNumber: 983
+  - type: enhancement
+    impact: minor
+    title: Replace insensitive terms with inclusive language
+    description: >-
```

Comment:
We need to change all `>-` to `|-`. Don't touch the existing ones to avoid conflicts.